ts/src/test/test.ts

```diff
@@ -26,11 +26,6 @@ const RequestTimeout = ccxt.RequestTimeout;
 // non-transpiled part, but shared names among langs
 class baseMainTestClass {
     info = false;
-    verbose = false;
```

Comment:
@ttodua now they might be used only once, but in the future, we might need to use them more than that, so I don't think it hurts having them as class members

Reply:
oky.